test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -2118,7 +2112,6 @@ TEST_P(BasicTest, BackendsRestart) {
   // yet having noticed that the backends were all down.
   CheckRpcSendFailure(CheckRpcSendFailureOptions().set_times(num_backends_));
   // Restart all backends.  RPCs should start succeeding again.
-  StartAllBackends();
```

Comment:
I think this change is incorrect.  We actually need to restart the backends here, since we stopped them above.

Reply:
You're right. I just did a blind removal of `StartAllBackends();` and hoped for the tests to catch any issues :)